Simplenote/Classes/UITextView+Simplenote.swift

```diff
@@ -157,19 +157,19 @@ extension UITextView {
     ///
     /// - Note: `contentInset.bottom` is expected to contain the bottom padding required by the keyboard. Capisce?
     ///
-    func editingRect() -> CGRect {
-        let paddingTop = safeAreaInsets.top
+    func editingRectInWindow() -> CGRect {
+        let paddingTop = frame.minY + safeAreaInsets.top
```

Comment:
That would only work if textView is added to the view that occupies full screen. I think the safest approach is to convert the original rect to window coordinates using UIView API.

Reply:
Updated, thank youu!!